diff --git a/gtk/gtkplacessidebar.c b/gtk/gtkplacessidebar.c
index d573f4a..baf4ea8 100644
--- a/gtk/gtkplacessidebar.c
+++ b/gtk/gtkplacessidebar.c
@@ -3052,6 +3052,29 @@ is_current_location_on_volume (GtkPlacesSidebar *sidebar,
   return current_location_on_volume;
 }
 
+static void 
+notify_noti_cb (GMountOperation *op,
+               char            *message,
+               gint64           time_left,
+               gint64           bytes_left,
+               gpointer         user_data)
+{
+    if (!bytes_left) {
+        gchar **strings;
+        strings = g_strsplit (message, "\n", 0);
+        GApplication *application = g_application_new ("hello.world", G_APPLICATION_FLAGS_NONE);
+           g_application_register (application, NULL, NULL);
+           GNotification *notification = g_notification_new (strings[0]);
+           g_notification_set_body (notification, strings[1]);
+           GIcon *icon = g_themed_icon_new ("media-removable");
+           g_notification_set_icon (notification, icon);
+           g_application_send_notification (application, NULL, notification);
+           g_object_unref (icon);
+           g_object_unref (notification);
+           g_object_unref (application);
+    }
+}
+
 static void
 do_unmount (GMount           *mount,
             GtkPlacesSidebar *sidebar)
@@ -3070,6 +3093,7 @@ do_unmount (GMount           *mount,
                                       NULL,
                                       unmount_mount_cb,
                                       g_object_ref (sidebar));
+      g_signal_connect (mount_op, "show-unmount-progress", notify_noti_cb, sidebar);
       g_object_unref (mount_op);
     }
 }
@@ -3244,6 +3268,7 @@ do_eject (GMount           *mount,
         g_drive_eject_with_operation (drive, 0, mount_op, NULL, drive_eject_cb,
                                       g_object_ref (sidebar));
     }
+  g_signal_connect (mount_op, "show-unmount-progress", notify_noti_cb, sidebar);
   g_object_unref (mount_op);
 }
 
