diff --git a/gtk/gtkfilechooserwidget.c b/gtk/gtkfilechooserwidget.c
index 45e4724..1b060a8 100644
--- a/gtk/gtkfilechooserwidget.c
+++ b/gtk/gtkfilechooserwidget.c
@@ -4080,7 +4080,7 @@ gtk_file_chooser_widget_map (GtkWidget *widget)
 
   settings_load (impl);
 
-  add_cwd_to_sidebar_if_needed (impl);
+//  add_cwd_to_sidebar_if_needed (impl);
 
   if (priv->operation_mode == OPERATION_MODE_BROWSE)
     {
