diff --git a/gtk/gtkwindow.c b/gtk/gtkwindow.c
index 699a702..45fb6bb 100644
--- a/gtk/gtkwindow.c
+++ b/gtk/gtkwindow.c
@@ -9179,11 +9179,11 @@ minimize_window_clicked (GtkMenuItem *menuitem,
                          gpointer     user_data)
 {
   GtkWindow *window = GTK_WINDOW (user_data);
-  GtkWindowPrivate *priv = window->priv;
+//  GtkWindowPrivate *priv = window->priv;
 
   /* Turns out, we can't iconify a maximized window */
-  if (priv->maximized)
-    gtk_window_unmaximize (window);
+//  if (priv->maximized)
+//    gtk_window_unmaximize (window);
 
   gtk_window_iconify (window);
 }
