diff --git a/gtk/gtkfontchooserwidget.c b/gtk/gtkfontchooserwidget.c
index ed52352..c01c348 100644
--- a/gtk/gtkfontchooserwidget.c
+++ b/gtk/gtkfontchooserwidget.c
@@ -616,7 +616,7 @@ gtk_font_chooser_widget_update_preview_attributes (GtkFontChooserWidget *fontcho
   attrs = pango_attr_list_new ();
 
   /* Prevent font fallback */
-  pango_attr_list_insert (attrs, pango_attr_fallback_new (FALSE));
+  pango_attr_list_insert (attrs, pango_attr_fallback_new (TRUE));
 
   /* Force current font and features */
   pango_attr_list_insert (attrs, pango_attr_font_desc_new (priv->font_desc));
