diff --git a/gtk/gtkpathbar.c b/gtk/gtkpathbar.c
index c626a3e..65f9ee4 100644
--- a/gtk/gtkpathbar.c
+++ b/gtk/gtkpathbar.c
@@ -109,7 +109,7 @@ struct _ButtonData
  * All buttons in front of a fake root are automatically hidden when in a
  * directory below a fake root and replaced with the "<" arrow button.
  */
-#define BUTTON_IS_FAKE_ROOT(button) ((button)->type == HOME_BUTTON)
+//#define BUTTON_IS_FAKE_ROOT(button) ((button)->type == HOME_BUTTON)
 
 G_DEFINE_TYPE_WITH_PRIVATE (GtkPathBar, gtk_path_bar, GTK_TYPE_CONTAINER)
 
@@ -1478,6 +1478,27 @@ make_directory_button (GtkPathBar  *path_bar,
   GtkWidget *child = NULL;
   ButtonData *button_data;
 
+  gchar *uri = g_file_get_uri (file);
+  gchar *home_uri = g_strdup_printf ("file://%s", g_get_home_dir ());
+  gchar *media_uri = g_strdup_printf ("file:///media/%s", g_get_user_name ());
+  const gchar *trash_uri = "trash:///";
+  const gchar *network_uri = "network:///";
+
+  if (!g_str_has_prefix (uri, home_uri) &&
+      !g_str_has_prefix (uri, media_uri) &&
+      !g_str_has_prefix (uri, trash_uri) &&
+      !g_str_has_prefix (uri, network_uri))
+    {
+      g_free (uri);
+      g_free (home_uri);
+      g_free (media_uri);
+      return NULL;
+    }
+
+  g_free (uri);
+  g_free (home_uri);
+  g_free (media_uri);
+
   file_is_hidden = !! file_is_hidden;
   /* Is it a special button? */
   button_data = g_new0 (ButtonData, 1);
@@ -1569,11 +1590,11 @@ gtk_path_bar_check_parent_path (GtkPathBar         *path_bar,
 	      ButtonData *button_data;
 
 	      button_data = list->data;
-	      if (BUTTON_IS_FAKE_ROOT (button_data))
-		{
-		  path_bar->priv->fake_root = list;
-		  break;
-		}
+//	      if (BUTTON_IS_FAKE_ROOT (button_data))
+//		{
+//		  path_bar->priv->fake_root = list;
+//		  break;
+//		}
 	    }
 	}
 
@@ -1694,10 +1715,13 @@ gtk_path_bar_get_info_callback (GCancellable *cancellable,
 				       file_info->first_directory, is_hidden);
   g_object_unref (file_info->file);
 
-  file_info->new_buttons = g_list_prepend (file_info->new_buttons, button_data);
+  if (button_data)
+    {
+      file_info->new_buttons = g_list_prepend (file_info->new_buttons, button_data);
 
-  if (BUTTON_IS_FAKE_ROOT (button_data))
-    file_info->fake_root = file_info->new_buttons;
+//      if (BUTTON_IS_FAKE_ROOT (button_data))
+//        file_info->fake_root = file_info->new_buttons;
+    }
 
   /* We have assigned the info for the innermost button, i.e. the deepest directory.
    * Now, go on to fetch the info for this directory's parent.
