From fe1e88e5628a8d97574442d24efe47e39fd5d26d Mon Sep 17 00:00:00 2001
From: donghun <donghun@gooroom.kr>
Date: Mon, 4 Nov 2019 11:12:23 +0900
Subject: [PATCH] Adding the patch to not add the current directory to the
 sidebar in GtkFileChooserDialog

---
 gtk/gtkfilechooserwidget.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkfilechooserwidget.c b/gtk/gtkfilechooserwidget.c
index d5a1791..4e9ef2f 100644
--- a/gtk/gtkfilechooserwidget.c
+++ b/gtk/gtkfilechooserwidget.c
@@ -4026,7 +4026,7 @@ gtk_file_chooser_widget_map (GtkWidget *widget)
 
   settings_load (impl);
 
-  add_cwd_to_sidebar_if_needed (impl);
+//  add_cwd_to_sidebar_if_needed (impl);
 
   if (priv->operation_mode == OPERATION_MODE_BROWSE)
     {
-- 
2.20.1

