From 522aec3e23e7d6dea3d9397360d36275757e0338 Mon Sep 17 00:00:00 2001
From: donghun <donghun@gooroom.kr>
Date: Mon, 4 Nov 2019 11:15:16 +0900
Subject: [PATCH] Adding the patch to disable entry completion popup in
 GtkFileChooserDialog

---
 gtk/gtkfilechooserentry.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkfilechooserentry.c b/gtk/gtkfilechooserentry.c
index 02f9d55..00571b0 100644
--- a/gtk/gtkfilechooserentry.c
+++ b/gtk/gtkfilechooserentry.c
@@ -743,7 +743,7 @@ refresh_current_folder_and_file_part (GtkFileChooserEntry *chooser_entry)
 
   folder_file = gtk_file_chooser_get_directory_for_text (chooser_entry, text);
 
-  set_completion_folder (chooser_entry, folder_file, dir_part);
+//  set_completion_folder (chooser_entry, folder_file, dir_part);
 
   if (folder_file)
     g_object_unref (folder_file);
-- 
2.20.1

