diff --git a/gtk/gtkfilechooserentry.c b/gtk/gtkfilechooserentry.c
index 02f9d55..00571b0 100644
--- a/gtk/gtkfilechooserentry.c
+++ b/gtk/gtkfilechooserentry.c
@@ -743,7 +743,7 @@ refresh_current_folder_and_file_part (GtkFileChooserEntry *chooser_entry)
 
   folder_file = gtk_file_chooser_get_directory_for_text (chooser_entry, text);
 
-  set_completion_folder (chooser_entry, folder_file, dir_part);
+//  set_completion_folder (chooser_entry, folder_file, dir_part);
 
   if (folder_file)
     g_object_unref (folder_file);
