diff --git a/gtk/gtkentry.c b/gtk/gtkentry.c
index 9223218..e459694 100644
--- a/gtk/gtkentry.c
+++ b/gtk/gtkentry.c
@@ -9643,7 +9643,7 @@ popup_targets_received (GtkClipboard     *clipboard,
       gtk_menu_shell_append (GTK_MENU_SHELL (menu), menuitem);
 
       if (info_entry_priv->show_emoji_icon ||
-          (gtk_entry_get_input_hints (entry) & GTK_INPUT_HINT_NO_EMOJI) == 0)
+          (gtk_entry_get_input_hints (entry) & GTK_INPUT_HINT_EMOJI) == 1)
         {
           menuitem = gtk_menu_item_new_with_mnemonic (_("Insert _Emoji"));
           gtk_widget_set_sensitive (menuitem,
